script.js

```diff
@@ -10,3 +10,215 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    supdescription: 'Keeping track of hundreds of components',
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 3,
+    title: 'Multi-Post Stories Gain+Glory',
+    snapshot: 'Images/assets/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    supdescription: 'Keeping track of hundreds of components',
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 4,
+    title: 'Multi-Post Stories Gain+Glory',
+    snapshot: 'Images/assets/Snapshoot-Portfolio.svg',
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    supdescription: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 5,
+    title: 'Multi-Post Stories Gain+Glory Gain+Glory',
+    snapshot: 'Images/assets/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    supdescription: 'Keeping track of hundreds of components',
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 6,
+    title: 'Multi-Post Stories Gain+Glory',
+    snapshot: 'Images/assets/Snapshoot-Portfolio.svg',
```

Comment:
- [ ] For a better user experience, Kindly make sure you are using the correct path on your images, as you may notice how your images are broken.
![image](https://github.com/newtonah0/Mobile-Portfolio/assets/97100091/c528d588-9411-4501-a258-3d7075df481b)